includes/class-cron.php

```
@@ -234,6 +221,32 @@ private function schedule_page_cron_jobs(): void {
+					if ( 0 === strpos( $page_url, $exclude_prefix ) ) {
+						return true;
+					}
+				}
+
+				if ( $page_url === $exclude_url ) {
+					return true;
+				}
+			}
+
+			return false;
+		}
```

PHPMD (2.15.0)

[error] The method is_url_excluded is not named in camelCase. (undefined)

(CamelCaseMethodName)